source/_ext/adventure_docs_extensions/__init__.py

```diff
@@ -23,13 +23,19 @@
 
 from pathlib import Path
 import re
+
+from docutils.statemachine import StringList
 from sphinx.application import Sphinx
 from docutils.parsers.rst import nodes
-
+from docutils.parsers.rst import Directive
 
 def setup(app: Sphinx):
     app.add_role("mojira", mojira_role)
     app.connect('html-collect-pages', _fix_cloudflare_name_mangling)
+    app.add_directive("kyori-dep", KyoriDepDirective)
+    app.add_config_value("api_version", "0.0.0", "html")
```

Comment:
Maybe make this a dict instead of hardcoding specific module names here? something like

```python
app.add_config_value("dep_versions", {'api': "0.0.0", 'platform': "0.0.0", 'platform_fabric': "0.0.0"}, "html", types=[dict])
```

just for future extendability

Reply:
Good idea!